app/helpers/request_helper.py

```diff
@@ -24,22 +42,27 @@ def remote_call(url, json=None):
         raise RetryableError("Connection error")
 
 
-def response_ok(response):
+def response_ok(response, service_url=None):
+    if service_url is not None:
```

Comment:
Should be written as ```if service_url``` as we probably want to check for other False values (such as an empty string or list). `is not None` literally checks that the object is not equal to the `None` singleton. `if a` will call `a.__nonzero__(self)`.

If the changes to `url_splitter` are made, this whole if else block can be removed.